3rdparty/openexr/CMakeLists.txt

```diff
@@ -27,9 +27,9 @@ set(ILMBASE_VERSION_PATCH "0")
 set(ILMBASE_VERSION "${ILMBASE_VERSION_MAJOR}.${ILMBASE_VERSION_MINOR}.${ILMBASE_VERSION_PATCH}")
 set(ILMBASE_VERSION_API ${ILMBASE_VERSION_MAJOR}_${ILMBASE_VERSION_MINOR})
 
-set(OPENEXR_VERSION_MAJOR "2")
-set(OPENEXR_VERSION_MINOR "3")
-set(OPENEXR_VERSION_PATCH "0")
+set(OpenEXR_VERSION_MAJOR "3")
```

Comment:
We are not going to maintain "own" build of OpenEXR anymore.
Use OpenEXR as external library.

See here:
https://github.com/opencv/opencv/pull/21324#issuecomment-1305925279

Reply:
got it, thanks!